HW/feature_eng.py

```diff
@@ -0,0 +1,61 @@
+import sys
+
+import matplotlib.pyplot as plt
```

Comment:
It would be a lot easier to do these plots in plotly. He has examples on the slides of how they should look and most of them even have a code example so you can recreate the exact same style easily.